README.md

```diff
@@ -4,32 +4,32 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
-| [IBM](https://www.theverge.com/2020/3/4/21165449/ibm-coronavirus-suspending-domestic-international-travel) | ? | Restricted | ? | ? | 2020-03-05 |
-| [Indeed](https://www.kvue.com/article/news/health/indeed-coronavirus-work-from-home/269-79c7797f-4d60-41df-bd9b-8b6bc25d9f3f) | Required | Restricted | Restricted | Restricted | 2020-03-04 |
-| Lyft | Encouraged | ? | ? | ? | 2020-03-04 | 
-| [Microsoft](https://www.theverge.com/2020/3/4/21164522/microsoft-coronavirus-response-comment-employees-memo-work-from-home) | Limited / Geo-specific | ? | ? | ? | 2020-03-03 |
-| [Nordstrom](https://www.geekwire.com/2020/coronavirus-live-updates-seattle-tech-community-grappling-covid-19/) | Encouraged | Restricted | Restricted | Restricted | 2020-03-04 |
-| [Redfin](https://www.seattletimes.com/business/some-seattle-tech-companies-tell-employees-to-work-from-home-to-slow-spread-of-coronavirus/) | Encouraged | ? | ? | ? | 2020-03-04 |
-| [Salesforce](https://www.salesforce.com/blog/2020/03/safety-and-wellbeing-those-around-you.html) | Encouraged | Restricted | Restricted | Restricted | 2020-03-03 |
-| Slack | Encouraged, Required in JP | Restricted | ? | Restricted | 2020-03-04 |
-| [Square](https://twitter.com/zamosta/status/1234658276781912064) | Encouraged | Restricted | Restricted | Restricted | 2020-03-03 |
-| [Stripe](https://stripe.com/newsroom/news/covid-19) | Encouraged, some Required | Restricted | Restricted | ? |2020-02-29 |
-| [Textio](https://www.seattletimes.com/business/some-seattle-tech-companies-tell-employees-to-work-from-home-to-slow-spread-of-coronavirus/) | Encouraged | ? | ? | ? | 2020-03-04 |
-| [Twitter](https://blog.twitter.com/en_us/topics/company/2020/keeping-our-employees-and-partners-safe-during-coronavirus.html) | Required | Restricted | Restricted | Restricted | 2020-03-03 |
-| [Typless](https://typless.com/2020/03/05/switching-to-fully-remote-work/) | Required | Restricted | Restricted | Restricted | 2020-03-04 |
-| Yelp | Encouraged | Restricted | ? | ? | 2020-03-03 |
+| Company    | WFH Policy                                  | Travel     | Visitors   | Events     | Updated    | Source                                                                                                                                                | Po  |
+| ---------- | ------------------------------------------- | ---------- | ---------- | ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| Amazon     | Encouraged, Recommended for Seattle to 3/31 | Restricted | Restricted | Restricted | 2020-03-04 | [businessinsider.com](https://www.businessinsider.com/companies-asking-employees-to-work-from-home-due-to-coronavirus-2020)                           |
+| Apple      | None                                        | Restricted | ?          | ?          | 2020-03-03 |
```

Comment:
I think we don't know if it's None? Unless there's a source I don't know about.

Reply:
Oh, I should have mentioned this. It's based on my own specific knowledge talking to Apple employees. I have no source or way to verify that.

Reply:
I've removed it.